Tiltfile

```diff
@@ -9,7 +9,7 @@ settings = {
     "deploy_cert_manager": True,
     "preload_images_for_kind": True,
     "kind_cluster_name": "capz",
-    "capi_version": "v0.3.7-rc.1",
+    "capi_version": "v0.3.7",
```

Comment:
made the change to upgrade capi version here so I can use the new clusterctl, will rebase when its merged in